api/Stream.cpp

```diff
@@ -190,10 +191,11 @@ float Stream::parseFloat(LookaheadMode lookahead, char ignore)
 
   if(isNegative)
     value = -value;
+
   if(isFraction)
-    return value * fraction;
-  else
-    return value;
+    value /= pow(10, digits_post_comma);
```

Comment:
`pow()` involves a logarithm and an exponential, which is very expensive on MMU-less processors.